src/style.css

```diff
@@ -0,0 +1,121 @@
+* {
+  padding: 0;
```

Comment:
why do we need to reset padding for `*` selector? reset only where needed

Reply:
after fixing the edits does not work correctly